eng/targets/Helix.Common.props

```diff
@@ -6,17 +6,16 @@
     <HelixQueueFedora34>(Fedora.34.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34-helix-20210924174119-4f64125</HelixQueueFedora34>
     <HelixQueueMariner>(Mariner)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-1.0-helix-20210528192219-92bf620</HelixQueueMariner>
     <HelixQueueArmDebian11>(Debian.11.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-arm64v8-20211001171229-97d8652</HelixQueueArmDebian11>
+
+    <!-- Do not attempt to override global property. -->
+    <RunQuarantinedTests Condition=" '$(RunQuarantinedTests)' == '' ">false</RunQuarantinedTests>
   </PropertyGroup>
 
-  <!-- Don't use <Choose/> for $(IsWindowsOnlyTest) checks. <When/> Condition is evaluated before project load. -->
-  <ItemGroup Condition=" '$(IsWindowsOnlyTest)' != 'true' ">
```

Comment:
To be clear: This worked but only because item group conditions are evaluated after the `msbuild` pass that sets all properties. It never worked in helix.proj and the item group is only used in Helix.targets.

I could have moved this `<ItemGroup/>` to Helix.props but decided to leave that file w/ only the bits that could only matter w/in our .csproj files or Helix.targets. `@(HelixAvailablePlatform)` is a fairly generic item group.